specviz/plugins/statistics/statistics_widget.py

```diff
@@ -57,17 +58,49 @@ def compute_stats(spectrum):
     spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
     region: `~specutils.utils.SpectralRegion`
     """
-    flux = spectrum.flux
-    mean = flux.mean()
-    rms = np.sqrt(flux.dot(flux) / len(flux))
-    return {'mean': mean,
-            'median': np.median(flux),
-            'stddev': flux.std(),
+
+    try:
+        cent = centroid(spectrum, region=None)  # we may want to adjust this for continuum subtraction
+    except Exception as e:
+        print("centroid err: ",e)
```

Comment:
Can you remove this print statement? Also, could you add all the exception output as logging errors?

```python
try:
    ...
except Exception as e:
    logging.error(e)
    ...